src/platform/imx8ulp/platform.c

```diff
@@ -0,0 +1,200 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2019 NXP
+//
+// Author: Daniel Baluta <daniel.baluta@nxp.com>
+
+#include <sof/compiler_info.h>
+#include <sof/debug/debug.h>
+#include <sof/drivers/edma.h>
+#include <sof/drivers/interrupt.h>
+#include <sof/ipc/driver.h>
+#include <sof/drivers/mu.h>
+#include <sof/drivers/timer.h>
+#include <sof/fw-ready-metadata.h>
+#include <sof/lib/agent.h>
+#include <sof/lib/clk.h>
+#include <sof/lib/cpu.h>
+#include <sof/lib/dai.h>
+#include <sof/lib/dma.h>
+#include <sof/lib/mailbox.h>
+#include <sof/lib/memory.h>
+#include <sof/lib/mm_heap.h>
+#include <sof/platform.h>
+#include <sof/schedule/edf_schedule.h>
+#include <sof/schedule/ll_schedule.h>
+#include <sof/schedule/ll_schedule_domain.h>
+#include <sof/sof.h>
+#include <sof/trace/dma-trace.h>
+#include <ipc/dai.h>
+#include <ipc/header.h>
+#include <ipc/info.h>
+#include <kernel/abi.h>
+#include <kernel/ext_manifest.h>
+#include <version.h>
+#include <errno.h>
+#include <stdint.h>
+
+struct sof;
+
+static const struct sof_ipc_fw_ready ready
+	__section(".fw_ready") = {
+	.hdr = {
+		.cmd = SOF_IPC_FW_READY,
+		.size = sizeof(struct sof_ipc_fw_ready),
+	},
+	/* dspbox is for DSP initiated IPC, hostbox is for host initiated IPC */
+	.version = {
+		.hdr.size = sizeof(struct sof_ipc_fw_version),
+		.micro = SOF_MICRO,
+		.minor = SOF_MINOR,
+		.major = SOF_MAJOR,
+#ifdef DEBUG_BUILD
+		/* only added in debug for reproducability in releases */
+		.build = SOF_BUILD,
+		.date = __DATE__,
+		.time = __TIME__,
```

Comment:
This was changed in 9f8cce15224, can you align?

The joys of copy/paste/diverge...